purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/BillingClientUseCase.kt

```diff
@@ -37,15 +37,16 @@ internal abstract class BillingClientUseCase<T>(
     }
 
     abstract fun executeAsync()
-    abstract fun onOk(received: T)
```

Comment:
Had to make some changes to the base class because queryPurchases chains two calls to the BillingClient